components/camel-robotframework/src/main/docs/robotframework-component.adoc

```diff
@@ -0,0 +1,160 @@
+[[Robotframework-component]]
+== Robotframework Component
+
+*Available as of Camel version 3.0.0*
+
+The *robotframework:* component allows for processing camel exchanges
+in acceptence test suites which are already implemented with its own DSL. 
+The depending keyword libraries that can be used inside test suites 
+implemented in Robot DSL, could have been implemented either via 
+Java or Pyhton.
+
+This component will let you execute business logic of acceptence test cases 
+in Robot language on which you can pass parameters to feed data via power 
+of Camel Routes, however there is no reverse binding of parameters back where
+you can pass values back into Camel exchange. Therefore, for that reason, it actually
+acts like a template language passing camel exchanges by binding data into the test 
+cases implemented. 
+
+[source,xml]
+-------------------------------------------------------------------------------------
+<dependency>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>camel-robotframework</artifactId>
+    <version>x.x.x</version> <!-- use the same version as your Camel core version -->
+</dependency>
+-------------------------------------------------------------------------------------
+
+### URI format
+
+[source,java]
+-------------------------------
+robotframework:templateName[?options]
+-------------------------------
+
+Where *templateName* is the classpath-local URI of the template to
+invoke; or the complete URL of the remote template (eg:
+\file://folder/myfile.robot).
+
+You can append query options to the URI in the following format,
+`?option=value&option=value&...`
+
+### Options
+
+// component options: START
+
```

Comment:
I think this must be populated anyway.

Reply:
i ran full build but those were not populated. doing it again. let me see if they will get populated. will update soon.

Reply:
Is not going to work I guess, I think there is a bug in the maven plugin, when it checks for the for the Schema JSON file that is generated from the `schema` endpoint parameter, it checks with case sensitivity, since your schema is `scheme = "RobotFramework"`, it failed to load the json file since it was expecting the name to be without uppercase `robotframework.json`. To workaround it, can you please change the schema name to `robotframework` and run the full build again? I will fix that small bug later

Reply:
Update: I was wrong here, in fact your schema name *should* be in small case